docs/search_vignette.ipynb

```diff
@@ -138,10 +131,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this example, there are two more added rows from \\_\\_find_add_srows. When the array goes into \\_\\_find_add_srows, the repeats are like this:\n",
+    "#FIX: Still thinking of how to deal with this, it's part of the private function\n",
```

Comment:
Request for suggestions

Reply:
the one thing I am confused by is if this example is for the `_find_add_srows` function - if it is, we could just get rid of the example that uses this function? Did we not agree to delete the documentation of the private functions from the vignette? Because the ones from the `assemble_vignette` are deleted

Reply:
Yeah so the thing is that I was about to delete this because it was part of the original `__find_add_srows` (which has been deleted and merged into `__find_add_rows`) and the writing also suggested so, because I didn't really work closely on this module. But when I checked the pictures folder, the name of the photos used in this chunk of documentation is labelled find_all_repeats... something which made me think that it's related to `find_all_repeats` which is a public function. So I'm a bit confused with how to deal with this part of the documentation

Reply:
I couldn't make up my mind on whether to delete the pictures or not, maybe the writing could be changed to fit `find_all_repeats` or something? I'll look more closely to this

Reply:
Ahh! Perhaps @bwang64 and @Chenhui-Jia can shed more light on this since they've worked on the `search` module more closely?